pkg/apis/compliance/v1alpha1/customrule_types.go

```diff
@@ -0,0 +1,114 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ScannerType string
+
+const (
+	ScannerTypeCEL      ScannerType = "CEL"
+	ScannerTypeOpenSCAP ScannerType = "OpenSCAP"
+	ScannerTypeUnknown  ScannerType = "Unknown"
```

Comment:
Based on the code above - we'd never use this, right?

Reply:
no, we will not use that until later in the reconciler loop

Reply:
Ok - so we will use it, just not now?